r/R/utilities_development.R

```diff
@@ -0,0 +1,11 @@
+#' Loads Experiment data
+#'
+#' @param exp_id
+#'
+#' @return
+#' @export
+#'
+#' @examples
+load_experiment_data <- function(exp_id = "test") {
+  return(readRDS(paste0("../data/", exp_id, "/r.rds", sep = "")))
```

Comment:
in this `worker/data` folder, under `test` I can only see `r.rds.gz`, not `r.rds`. How does it pick the `r.rds.gz` file then?